texas_death_row.qmd

```diff
@@ -29,19 +31,44 @@ Now that we have the data loaded, lets answer some questions.
 
 
 ```{r question 1}
+TXDR %>%
+    count(County) %>%
+    arrange(desc(n)) 
 
 ```
 
+Q1: By far the highest county is Harris County with 128 exections, followed by Dalls County with less than half of that, at 58.
 
 ```{r question 2}
+ClaimsInnocence <- na.omit(str_detect(TXDR$`Last Statement`, "innocent|innocence"))
+
+sum(ClaimsInnocence == TRUE) / sum(ClaimsInnocence == FALSE)
 
 ```
 
-```{r question 3}
+Q2: 33 of the 410 inmates who provided statements claimed to be innocent, based on a search of the keywords "innocent" and "innocence"
 
+```{r question 3}
+TXDR$`Execution Date` <- as.Date(TXDR$`Execution Date`, format="%d/%m/%Y")
+tab <- table(cut(TXDR$`Execution Date`, 'year'))
+dframe <- data.frame(Date=format(as.Date(names(tab)), '%Y'),
+           Frequency=as.vector(tab))
+
+YearsOfExecution <-ggplot(dframe, aes(x=Date, y=Frequency)) +
+geom_bar(stat="identity", fill="#56CBF9", color="black") + scale_x_discrete(guide = guide_axis(n.dodge = 3)) + labs(x="Year", title = "Exectutions Over Time") + theme(panel.background = element_rect(fill="#F0F0C9"), panel.grid.major= element_line(colour="#747274"), panel.grid.minor=element_line(colour="#747274"), plot.background=element_rect(fill="#F0F0C9"),
+        plot.title = element_text(face="bold", size=14, hjust=.5),
+        plot.subtitle = element_text(hjust=.5))
+YearsOfExecution
```

Comment:
I really like the creativeness of the plot and your color scheme! Here you don't have to assign the plot to an object and call the object. You can just leave the `ggplot()` call without the assignment